src/modules.js/showDb.js

```diff
@@ -0,0 +1,41 @@
+export const showDb = (db) => {
+  const container = document.getElementById('list-item-container');
+
+  container.innerHTML = '';
+
+  db.forEach((element, index) => {
+    let check = '';
+    if (element.completed) {
+      check = 'checked';
+      // ArrayOfActivities[index].style.textDecoration = "line-through";
```

Comment:
- [ ] To keep your repo clean , kindly remove any commented code from your project 🙏

Reply:
done